src/database/queries/saveEvent.js

```diff
@@ -0,0 +1,3 @@
+const Events = require('../events')
+
+module.exports = (object) => new Events({ object }).save();
```

Comment:
Good, you've created a query, and used it where?

Reply:
event.js in controller

Reply:
No you haven't required it in `event.js`.